src/lib.rs

```diff
@@ -4282,14 +4279,15 @@ pub unsafe extern "C" fn wgpuDevicePoll(
 
     let maintain = match wait {
         true => match submission_index {
-            Some(index) => wgt::Maintain::WaitForSubmissionIndex(*index),
-            None => wgt::Maintain::Wait,
+            Some(index) => wgt::PollType::WaitForSubmissionIndex(*index),
+            None => wgt::PollType::Wait,
         },
-        false => wgt::Maintain::Poll,
+        false => wgt::PollType::Poll,
     };
 
     match context.device_poll(device_id, maintain) {
-        Ok(queue_empty) => queue_empty,
+        // TODO check if we need to check the PollStatus to return true or false
+        Ok(_) => true,
```

Comment:
yes, this needs to match if PollStatus == QueueEmpty

Reply:
Should we log something when it does not match ?

Reply:
I mean, `matches!(poll_status, QueueEmpty)` instead of always true

Reply:
I get it, but should we add a log to explain what is happening when we return false ?

Reply:
fixed